feat(content): add Republic Navy Advisory System and add related humanitarian missions news (#8871)
endless-sky/endless-sky: data/human/jobs.txt

		payment 8500 80
		dialog "You drop off your cargo of <commodity> and collect your payment of <payment>. A group of <planet> citizens thank you for your assistance and bid you farewell."

mission "FW Wildfire Aid [2]"
	job
	repeat

> 	name "Firefighters to <planet>"

Author: Fixed

		system
			distance 5 5
		fleet "Marauder fleet X"

mission "Republic Navy Advisory System [frozen]"
	name "Cold Relief"

> 	description "Humanitarian aid of <cargo> to <destination> by <date> following a severe cold snap."

Reviewer: ```suggestion
	description "Deliver humanitarian aid of <cargo> to <destination> by <date> following a severe cold snap. Payment is <payment>."
```

	minor
	repeat
	deadline
	source

> 		government "Republic"

Reviewer: ```suggestion
		government "Republic" "Syndicate"
```
Syndicate planets are part of the Republic too and get Navy protection, so they'd presumably help out.

	destination

> 		government "Republic" "Syndicate" "Independent" "Neutral"

Reviewer: ```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

		attributes "frozen"
	substitutions
		"<problem>" "cold snap"
		"<problem>" "freeze"
			"random" < 60
		"<problem>" "snowstorm"
			"random" < 30
		"<problem>" "blizzard"
			"random" < 10
	cargo "cold aid" 10 15 0.5
	to offer
		random < 1
	on offer
		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
	on complete
		payment 8000

> 		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most.`

Reviewer: ```suggestion
		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`
```

mission "Republic Navy Advisory System [volcanic]"
	name "Volcanic Relief"

> 	description "Humanitarian aid of <cargo> to <destination> by <date> following a severe volcanic eruption."

Reviewer: ```suggestion
	description "Deliver humanitarian aid of <cargo> to <destination> by <date> following a severe volcanic eruption. Payment is <payment>."
```

	minor
	repeat
	deadline
	source

> 		government "Republic"

Reviewer: ```suggestion
		government "Republic" "Syndicate"
```

	destination

> 		government "Republic" "Syndicate" "Independent" "Neutral"

Reviewer: The way you have these missions set up, you can get unnecessarily long trips to your destination. One possible mission might have a player go from Wayfarer to Calda and and get only 8000 credits for it. You should consider adding a distance filter up to 8 systems away.
```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

		attributes "volcanic" "lava"
	substitutions
		"<problem>" "volcanic event"
		"<problem>" "volcanic eruption"
			"random" < 60
		"<problem>" "gas eruption"
			"random" < 30
	cargo "flames aid" 10 15 0.5
	to offer
		random < 1
	on offer
		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
	on complete
		payment 8000

> 		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most.`

Reviewer: ```suggestion
		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`
```
(Consider changing this suggestion to something more appropriate)

mission "Republic Navy Advisory System [wildfire cargo]"
	name "Wildfire Relief"

> 	description "Humanitarian aid of <cargo> to <destination> by <date> following a severe wildfire."

Reviewer: ```suggestion
	description "Deliver humanitarian aid of <cargo> to <destination> by <date> following a severe wildfire. Payment is <payment>."
```

	minor
	repeat
	deadline
	source
		government "Republic"
	destination

> 		government "Republic" "Syndicate" "Independent" "Neutral"

Reviewer: ```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

		attributes "forest"
	substitutions
		"<problem>" "forest fire"
		"<problem>" "brush fire"
			"random" < 60
		"<problem>" "wildfire"
			"random" < 30
	cargo "flames aid" 10 15 0.5
	to offer
		random < 1
	on offer
		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
	on complete
		payment 8000

> 		dialog `Ash-covered, but relieved aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most.`

Reviewer: ```suggestion
		dialog `Ash-covered, but relieved aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`
```
Ditto

mission "Republic Navy Advisory System [wildfire firefighters]"
	name "Transport Firefighters"

> 	description "Bring <bunks> firefighters to <planet> by <date>."

Reviewer: ```suggestion
	description "Transport <bunks> firefighters to <planet> by <date>. Payment is <payment>."
```

	minor
	repeat
	deadline
	source

> 		government "Republic"

Reviewer: ```suggestion
		government "Republic" "Syndicate"
```

	destination

> 		government "Republic" "Syndicate" "Independent" "Neutral"

Reviewer: ```suggestion
		government "Republic" "Syndicate" "Independent" "Neutral"
		distance 1 8
```

		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
	on complete
		payment 8000

> 		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`

Reviewer: ```suggestion
		dialog `Shivering, yet appreciative aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've received <payment> for your service.`
```

		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
	on complete
		payment 8000

> 		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`

Reviewer: ```suggestion
		dialog `Relieved-looking aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've received <payment> for your service.`
```
Oops.

		dialog `This is the Republic Navy Advisory System: <planet> is experiencing a severe <problem>. Volunteers are requested to supply <cargo> to <planet> by <date>. A stipend of <payment> is provided to cover fuel, crew, and any incidental expenses.`
	on complete
		payment 8000

> 		dialog `Ash-covered, but relieved aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've recieved <payment> for your service.`

Reviewer: ```suggestion
		dialog `Ash-covered, but relieved aid workers welcome your shipment. You look on, feeling satisfied that <cargo> will soon arrive to those that need it most. Back in your ship, you're notified that you've received <payment> for your service.`
```
